docs/cash-savings/Interest.md

```diff
@@ -0,0 +1,31 @@
+---
+tags: [Cash Savings]
+---
+
+# Interest
+
+## How interest is paid
+
+Interest is paid on the 1st of each month on an accrual basis. This means that interest is calculated (accrued) daily, but paid monthly. 
+
+## Interest rates
+
+Interest rates can be configured per product, split between the investor and the tenant, up to a maximum of the current [Bank of England base rate](https://www.bankofengland.co.uk/monetary-policy/the-interest-rate-bank-rate).
+
+## Rate changes
+
+Sometimes the Bank of England changes the base rate. When this happens we will use the most up to date figure and calculate the accrual for that day onwards accordingly. 
+
+## Cut offs
+
+An internal transfer must have been requested before the cut off in order to be eligible for interest. After this time interest will not be accrued until the next working day.
+
+## Withdrawals
+
+If cash is moved out of the savings portfolio, or completely withdrawn from the platform, interest will be paid as usual on the 1st of the next month, accrued for the days which it was held in the savings portfolio.
+
+For example, if £100 was held in a savings portfolio from 2025-01-01, and a transfer back to the wallet completes on 2025-01-15, 14 days worth of interest would be paid into the savings portfolio on 2025-02-01.
+
+## Fractional interest
+
+If a fractional amount of interest is accrued, i.e. not whole pennies, we will carry the interest forward to the next day. Similarly, if the total accrued interest at the end of the month is fractional, we will store that fraction and add it into next month's payout.
```

Comment:
Should we reword this to say we will round interest to the nearest penny each month and any fractional remainder is rolled over?